ssl/record/rec_layer_s3.c

```diff
@@ -387,7 +387,8 @@ int ssl3_write_bytes(SSL *s, int type, const void *buf_, size_t len,
      * If we are supposed to be sending a KeyUpdate then go into init unless we
      * have writes pending - in which case we should finish doing that first.
```

Comment:
Update this comment to say we also go into init if we have pending tickets to send?